packages/ETLoad.package/ETLPlaceholderObject.class/instance/loadIndexablePointerFieldsObjectUsing..st

```diff
@@ -0,0 +1,12 @@
+loading
+loadIndexablePointerFieldsObjectUsing: oopMap
+	| fieldCount object |
+	
+	"Object size includes base header word"
+	fieldCount := self header objectSizeInWords - 1.
+	object := self objectClass basicNew: fieldCount.
+	1 to: fieldCount do: [ :i |
+		object
+			instVarAt: i
+			put: (oopMap resolve: self bodyStream next ifInvalid: [ nil. ]). ].
```

Comment:
[This comment](https://github.com/bertfreudenberg/SqueakJS/blob/master/vm.js#L1551) explains the need for this. I will refactor the code and include an explanation.